doc/classes/PhysicsServer2D.xml

```diff
@@ -58,7 +58,7 @@
 			<return type="RID">
 			</return>
 			<description>
-				Creates an [Area2D].
+				Creates an [Area2D]. After creating an [Area2D] with this method, assign it with a space using [method area_set_space] to use the created [Area2D] in the physics world.
```

Comment:
It still says `assign it with a space` instead of `assign it to a space`.

Reply:
sure